Corona-Warn-App/src/main/res/layout/contact_diary_edit_persons_fragment.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+        <variable
+            name="viewModel"
```

Comment:
not sure if i like the view model injection, maybe use the `UIState` data class pattern instead, but fine for an rc i think